assembly/plaintime.ts

```diff
@@ -64,4 +238,86 @@ export class PlainTime {
 
     return ord(a.nanosecond, b.nanosecond);
   }
+
+  add<T>(durationToAdd: T): PlainTime {
+    const duration =
+      durationToAdd instanceof DurationLike
+        ? durationToAdd.toDuration()
+        : // @ts-ignore TS2352
+          (durationToAdd as Duration);
```

Comment:
```suggestion
        // @ts-ignore TS2352
        : (durationToAdd as Duration);
```

Reply:
Ohh crap, I should disable eslint in vscode 😅